sever-how-to/Server-Setup.md

```diff
@@ -70,7 +70,7 @@ For the installation and configuration steps follow the specific guides.
 1. [https://hub.docker.com/r/iceflower/redeclipse-server](https://hub.docker.com/r/iceflower/redeclipse-server)
 
 ## Configuration Files
-The configuration files are written in [cubescript](.md), lines starting with `//` are ignored. If you want to configure a setting simply remove the `//` in front of it and change it's value.
+The configuration files are written in cubescript, lines starting with `//` are ignored. If you want to configure a setting simply remove the `//` in front of it and change it's value.
```

Comment:
`s/it's/its/`

Reply:
there is not "cubescript.md" file, and I don't know cubescript so I can't write that article